app.json

```diff
@@ -6,7 +6,7 @@
     "orientation": "portrait",
     "icon": "./assets/icon.png",
     "splash": {
-      "image": "./assets/splash.png",
+      "image": "./assets/splash.jpg",
```

Comment:
why ? the png file is better than jpg for the alpha channel. Moreover I guess that iOS need png format